fn_query_tor_network/fn_query_tor_network/util/config.py

```diff
@@ -0,0 +1,20 @@
+# (c) Copyright IBM Corp. 2010, 2018. All Rights Reserved.
+# -*- coding: utf-8 -*-
+
+"""Generate a default configuration-file section for fn_query_tor_network"""
+
+from __future__ import print_function
+
+
+def config_section_data():
+    """Produce the default configuration section for app.config,
+       when called by `resilient-circuits config [-c|-u]`
+    """
+    config_data = u"""[fn_query_tor_network]
+base_url = https://onionoo.torproject.org/details
+#The Flag can be 'Running','Exit'
```

Comment:
what does running and exit mean? Maybe add a link to the onionoo page which describes it.

Reply:
changes are incorporated.